lib/src/components/GenericGeneSeqPanel.tsx

```diff
@@ -18,8 +19,8 @@ export default function GenericGeneSeqPanel(props: {
   const [result, setResult] = useState<Feature[]>();
   const [error, setError] = useState<unknown>();
   const [transcript, setTranscript] = useState<Feature>();
-  const [mode, setMode] = useState("gene");
   const feature = transcript || result?.[0];
+  const [model] = useState(SequenceFeatureDetailsF().create({}));
```

Comment:
Important part here: we instantiate a whole "sequence feature detail model type" which controls the state of the component. this is a mobx-state-tree model, and it stores the "mode" type internally